App.js

```diff
@@ -1,24 +1,80 @@
-import React from 'react'
+import React, { useState } from 'react'
 import styled from 'styled-components/native'
 
 const Container = styled.View`
   flex: 1;
-  background-color: papayawhip;
-  justify-content: center;
   align-items: center;
+  background: papayawhip;
 `
 
-const Title = styled.Text`
-  font-size: 24px;
-  color: palevioletred;
+const Card = styled.Image`
+  flex: 1;
+  width: 200px;
+  height: 200px
+  resizeMode: contain;
+  margin: 5px
+`
+const Text = styled.Text`
+  align-items: center;
+  justify-content: center;
+  color: white;
+`
+const FetchButton = styled.TouchableOpacity`
+  background-color: tomato;
+  align-items: center;
+  justify-content: center;
+  fontWeight: bold;
```

Comment:
We shouldn't use camelCase in styled component, `font-weight` is proper way

Reply:
changed all camelCase to kebab-case